email-report.html

```diff
@@ -1,31 +1,5 @@
-18:45:02,601 |-INFO in ch.qos.logback.classic.LoggerContext[default] - Could NOT find resource [logback-test.xml]
```

Comment:
I don't believe this file should exist in this commit - it looks like test data that was accidentally committed.

Reply:
I committed this file because I saw it already existing in the repo. But definitely can be taken out